testsuite/src/it/java/org/graalvm/tests/integration/utils/versions/GraalVersionProperty.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021, Red Hat Inc. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.graalvm.tests.integration.utils.versions;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Sets the property specified by {@code name} (default 'graal.vm.version')
+ * to the currently in-use Graal version for the annotated test run
```

Comment:
@jerboaa do I understand correctly that this implies that `$JAVA_HOME` is the same as `$GRAALVM_HOME`? Will this still work if we use OpenJDK to run JUNIT and only use GraalVM for the `native-image` part?

Reply:
I'm not sure, but don't think this is the implication. It seems to assume `native-image` is on the `PATH`, though. It uses [UsedVersion]( https://github.com/Karm/mandrel-integration-tests/blob/master/testsuite/src/it/java/org/graalvm/tests/integration/utils/versions/UsedVersion.java#L36) which is being used elsewhere to determine the GraalVM version (e.g. for the `IfMandrelVersion` annotation). See here:
https://github.com/Karm/mandrel-integration-tests/pull/111/files#diff-e42933e425be03ad47f5f5ede23cae81457941944ed68b35a612ceb10d45c314R38

Does that answer your question?

Reply:
Yes, thanks @jerboaa.